modules/www/main.tf

```diff
@@ -407,28 +422,51 @@ resource "aws_ecs_task_definition" "www_task_definition" {
        "privileged": false,
        "name": "nerves_hub_www",
        "environment": [
-         {
-           "name": "ENVIRONMENT",
-           "value": "${terraform.workspace}"
-         },
-         {
-           "name": "APP_NAME",
-           "value": "nerves_hub_www"
-         }
+         ${local.ecs_shared_env_vars}
        ],
+       "volumesFrom": [],
+       "mountPoints": [],
        "logConfiguration": {
-         "logDriver": "awslogs",
+         "logDriver": "awsfirelens",
          "options": {
-           "awslogs-region": "${var.region}",
-           "awslogs-group": "${var.cluster.log_group_name}",
-           "awslogs-stream-prefix": "nerves_hub_www"
-         }
+            "Name": "datadog",
+            "compress": "gzip",
+            "Host": "http-intake.logs.datadoghq.com",
+            "dd_service": "${local.app_name}",
+            "dd_source": "elixir",
+            "dd_message_key": "log",
+            "dd_tags": "env:${var.environment_name},application:${local.app_name}-${var.environment_name},version:${var.docker_image}",
+            "TLS": "on",
+            "provider": "ecs"
+          },
+          "secretOptions": [
+            {
+              "name": "apikey",
+              "valueFrom": "${module.firelens_log_config.datadog_key_arn}"
+            }
+          ]
        }
-     }
+     },
+     ${module.firelens_log_config.datadog_container}
    ]
-
 DEFINITION
 
+depends_on = [
+    module.firelens_log_config
+  ]
+
+}
+
+module "firelens_log_config" {
```

Comment:
We might need another one of these for the datadog log_config like this: https://github.com/smartrent/terraform-aws-ecs-container_definitions/blob/eefdc7d52f1d43621c9380a6df266d66fc444ffa/firelens.tf#L41

The datasource is different: https://github.com/smartrent/terraform-aws-ecs-container_definitions/blob/eefdc7d52f1d43621c9380a6df266d66fc444ffa/firelens.tf#L49

I'm not sure if there's a way to have that just in the firelens_log_config. 🤔 We'll want to be able to change these vars if needed for the dd container as well. 🤔